### ISSUE TITLE
One step install not working for **ME**

### ISSUE THREAD

**Issue by osok** [oobabooga/text-generation-webui#4169]
### Describe the bug

TLDR:
I'm not familar with miniconda and exactly how it works.  However it clearly is not working when using start_linux.sh and I think its why the one step is broke.  While I call out a few problems, the issues I'm reporting is that the start_linux.sh is not working/

Symptoms:
I have been using TextGen WebUI for a few months now.  I have orginally been building on AWS EC2 with NVIDIA GPUs, G5.4xlarge and g5.48xlarge.  I also have a Computer at home with 2 4090's.  I have been able to succesfully get things working, I have created a set of instructions that seem to work   Last night and today things stop working.  I update and things break.

One environment (AWS) I got TextGen reinstalled by creating a new conda env and downloading new textgen and manually installing but when I go to the model screen I cannot load any models even though there are like 10 downloaded.

One my local envirnment TextGen worked a few weeks ago, then today it didnt recognize the GPU's. I updated to the latest code and it ran but no models worked.  errors appeared to indicate no CUDA support was available. 

Today I tried to use it and it loaded a TheBloke/Wizard-Vicuna-30B-Uncensored-GPTQ, it loaded ok, then puked  out errors related to CUDA not existing. I don't have the errors, I"m about 4 itterations past trying to fix that.  I mention it becasue this is what I did:

- I remove anaconda3 from my computer
- exited the ssh session 
- re entered ssh
- clone the current textgen repo
- tried to use the one step installer  start_linux.sh

This is what I got:
It downloads miniconda
appears to be creating an environment

`
Downloading and Extracting Packages

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate /home/michael/ai/textgen2/installer_files/env
#
# To deactivate an active environment, use
#
#     $ conda deactivate

*******************************************************************
* WARNING: You haven't downloaded any model yet.
* Once the web UI launches, head over to the "Model" tab and download one.
*******************************************************************

Traceback (most recent call last):
  File "/ai/michael/textgen2/server.py", line 5, in <module>
    from modules.block_requests import OpenMonkeyPatch, RequestBlocker
  File "/ai/michael/textgen2/modules/block_requests.py", line 4, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
`

If I try to run

`conda activate /home/michael/ai/textgen2/installer_files/env
`
it fails to find conda installed.  If I exit and reenter ssh it desn't work either.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. I removed Anaconda3
2. I cloned the repo
3. I ran start_linux.sh

It appears to download miniconda
it appears to create a conda env
then

`
Downloading and Extracting Packages

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate /home/michael/ai/textgen2/installer_files/env
#
# To deactivate an active environment, use
#
#     $ conda deactivate

*******************************************************************
* WARNING: You haven't downloaded any model yet.
* Once the web UI launches, head over to the "Model" tab and download one.
*******************************************************************

Traceback (most recent call last):
  File "/ai/michael/textgen2/server.py", line 5, in <module>
    from modules.block_requests import OpenMonkeyPatch, RequestBlocker
  File "/ai/michael/textgen2/modules/block_requests.py", line 4, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
`

If I try to run

`conda activate /home/michael/ai/textgen2/installer_files/env
`
it fails to find conda installed.  If I exit and reenter ssh it desn't work either.

### Logs

```shell
Traceback (most recent call last):
  File "/ai/michael/textgen2/server.py", line 5, in <module>
    from modules.block_requests import OpenMonkeyPatch, RequestBlocker
  File "/ai/michael/textgen2/modules/block_requests.py", line 4, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

### System Info

```shell
Environment: 
I have (had) anaconda installed.  Typically I use python 3.10.9 in my environments.  some environments like training required older versions.

I do use the environments for other things like Jupyter, and Langchain.  I'll pick one environment and describe.  I did try to get tensorflow-gpu running in one of my anaconda environments.

Local Computer.  I have an AMD 7950X with 64 GB ram two water cooled 4090.  Ubuntu 22.04.  CUDA Drivers and toolkit installed.

` uname -a
Linux evos 6.2.0-33-generic #33~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Sep  7 10:33:52 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
`

` nvidia-smi
Tue Oct  3 18:53:59 2023
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 530.30.02              Driver Version: 530.30.02    CUDA Version: 12.1     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                  Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf            Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 4090         On | 00000000:01:00.0 Off |                  Off |
|  0%   33C    P8               32W / 480W|    183MiB / 24564MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
|   1  NVIDIA GeForce RTX 4090         On | 00000000:03:00.0 Off |                  Off |
|  0%   32C    P8               32W / 480W|      6MiB / 24564MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+

+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|    0   N/A  N/A      1659      G   /usr/lib/xorg/Xorg                          167MiB |
|    0   N/A  N/A      1844      G   /usr/bin/gnome-shell                         14MiB |
|    1   N/A  N/A      1659      G   /usr/lib/xorg/Xorg                            4MiB |
+---------------------------------------------------------------------------------------+
`

`nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2023 NVIDIA Corporation
Built on Mon_Apr__3_17:16:06_PDT_2023
Cuda compilation tools, release 12.1, V12.1.105
Build cuda_12.1.r12.1/compiler.32688072_0
`


At one point I had anaconda installed with an environment for 
- TextGen
- Langchain (which I used with Jupyter
- Jupyter with Tensorflow
- fastai
-and a few others

I think the installation of tensorflow messeed my testgen environment.  At one point I could use both GPU and load 30B models and even Falcon 40B with adding some CPU.
```

**Comment by osok** [comment]
I deleted everything and tried again. and its building.
